### PR TITLE
Create IPerson.cs

### PR DESCRIPTION
**Pull request by RaulGlzChubb**
Add IPerson.cs to main

### CODE REVIEW

File: IPerson.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+public interface ISampleInterface
```
Comment (isaacjhonatan) [inline IPerson.cs]: Falta el namespace

Comment (RaulGlzChubb) [reply IPerson.cs]: Namespace agregado

**Review by isaacjhonatan — APPROVED** [review]
Fixed.

**Comment by RaulGlzChubb** [comment]
Merge changes from commit "Create IPerson.cs" to main.

**Comment by RaulGlzChubb** [comment]
Reopen pull request

**Comment by RaulGlzChubb** [comment]
Merge changes from commit "Create IPerson.cs" to main successfull

**Pull request merged**